app/api/group/create/route.ts

```diff
@@ -0,0 +1,18 @@
+import {createGroup} from '@/service/pads/GroupManager';
+import { APIResponse } from '@/types/APIResponse';
+import { NextResponse } from "next/server";
```

Comment:
```suggestion
import { NextResponse } from "next/server";
import {createGroup} from '@/service/pads/GroupManager';
import { APIResponse } from '@/types/APIResponse';
```


put node internal, npm package, our code